packages/create-plugin/README.md

```diff
@@ -20,7 +20,7 @@ Create Grafana plugins with ease.
 
 - [Plugin Tools docs](https://grafana.github.io/plugin-tools/)
 - [Plugin developer docs](https://grafana.com/docs/grafana/latest/developers/plugins/)
-- [Plugin migration guide](https://grafana.com/docs/grafana/latest/developers/plugins/migration-guide/)
+- [Plugin migration guide](../../docusaurus/docs/migration-guide/)
```

Comment:
Should we point to the live site rather than directories in the repo?

```suggestion
- [Plugin migration guide](https://grafana.github.io/plugin-tools/docs/migration-guide/)
```

Reply:
I think when the site is published on the dev portal we will need to have relative links, no? Links to the site root would probably all need to be changed.